crates/node/p2p/src/gossip/gater.rs

```diff
@@ -192,12 +225,14 @@ impl ConnectionGate for ConnectionGater {
         self.blocked_addrs.iter().cloned().collect()
     }
 
-    fn block_subnet(&mut self, _subnet: &str) {
-        // TODO
+    fn block_subnet(&mut self, subnet: &str) {
+        self.blocked_subnets.insert(subnet.to_owned());
+        debug!(target: "gossip", ?subnet, "Blocked subnet");
```

Comment:
Then, if the subnet cannot be parsed, we can just emit a warning here without needing to error.